src/modelbench/run.py

```diff
@@ -70,6 +73,7 @@ def cli() -> None:
 )
 @click.option("--max-instances", "-m", type=int, default=100)
 @click.option("--debug", default=False, is_flag=True)
+@click.option("--machine-reports", default=False, is_flag=True, help="Print only machine-readable progress reports")
```

Comment:
Yeah I agree it’s a little awkward. What would be the difference between log and structured? Maybe it would be simpler to just change the name to `--structured_progress_reports`, what do you think?

> Relatedly, is a boolean debug option sufficiently granular? It means you either get all the logged output or none

If neither --debug nor --machine-reports (or whatever) is set, you would still get the INFO level logs.

Reply:
Structured could be json, log could be one line per event. But it sounds like that hasn't been necessary, so it's fine as is.